package-lock.json

```diff
@@ -1,17604 +1,8 @@
 {
   "name": "public-browse",
   "version": "2.1.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
While I can't imagine the downgrade to v1 is going to cause any issues it might be best to use the latest version of npm (v7.x) to avoid the downgrade. I think we're using v2 in most of our repos now so it would be better from a consistency point-of-view.